src/main/java/frc/robot/swerve/ModuleIOTalonFX.java

```diff
@@ -89,6 +117,27 @@ public void updateInputs(SwerveModuleInputs inputs) {
 
         inputs.moduleDistance = getModulePosition().distanceMeters;
         inputs.moduleState = getModuleState();
+
+        List<Double> distanceList = distanceQueue.stream().toList();
+        int nD = distanceList.size();
+        inputs.highFreqDistances = new double[nD];
+        for (int i = 0; i < nD; i++) {
+            inputs.highFreqDistances[i] = Units.rpsToMetersPerSecond(
+                    distanceList.get(i), SwerveConstants.WHEEL_DIAMETER/2
+            );
+        }
+        distanceQueue.clear();
+
+        List<Double> angleList = angleQueue.stream().toList();
+        int nA = angleList.size();
+        inputs.highFreqAngles = new double[nA];
+        for (int i = 0; i < nA; i++) {
+            inputs.highFreqAngles[i] = Units.rpsToRadsPerSec(angleList.get(i));
+        }
+        angleQueue.clear();
+
+        driveMotor.getConfigurator().apply(driveConfig.Slot0);
+        angleMotor.getConfigurator().apply(angleConfig.Slot0);
```

Comment:
I'm sorry, I just have two last little things.
1. It's not a very good idea to update the PID inside the motor every loop. You need to use the `hasChanged` method in the `LoggedTunableNumber` object.
2. The `Slot0` object is never updated, so the PID would never really change.